### PR TITLE
Pagination markup kit

### PR DESCRIPTION
**Pull request by antelopeb**
First shot. Pulled markup over from the old component, and I'm sure it needs some accessibility help.

### CODE REVIEW

**Comment by antelopeb** [comment]
@dengeist & @pauljadam -- I introduced a non-standard behavior when I introduced focus management. The focus stays on the clicked button, but it forces Chrome to put visual focus on a button that was clicked with a mouse vs a keyboard action. Is that something I should try and differentiate by following the keypress events, too?

Also, let me know if I need to add any arrow key navigation or otherwise. Right now all items are treated as separate buttons that are tabbable, but there is no focus trapping or otherwise.

File: pagination/markup-kit/js/main.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
+'use strict';
```
Comment (dengeist) [inline pagination/markup-kit/js/main.js]: `'use strict'` should be _inside_ our iifes :)

Comment (antelopeb) [reply pagination/markup-kit/js/main.js]: Agreed and changed. Should we change in markup kit too so we don't have a repeated error?

**Comment by dengeist** [comment]
Yeah, I think we should change these to links. I also agree about the more descriptive labels for the individual links. I didn't notice that the ellipsis was missing a label but it definitely needs one!

File: pagination/markup-kit/js/main.js (hunk 1 above)
Comment (davodey) [reply pagination/markup-kit/js/main.js]: yes yes yes!

Comment (dengeist) [reply pagination/markup-kit/js/main.js]: @davodey @antelopeb working on an overhaul to markup-kit atm; thanks for highlighting this issue

**Comment by antelopeb** [comment]
@pauljadam & @dengeist -- should I just change buttons to links, then? Or should I allow some type of configuration to switch between the two?

Full disclosure, the second option worries me because people would struggle with that I think... but I'm open to thoughts.

**Comment by pauljadam** [comment]
@antelopeb  I think it's fine to change them to <a href> links rather than <button>. The only issue is that you may have to do a JavaScript preventDefault() to stop the default behavior of the browser scrolling to the top if you have an <a href="#"> link. You could also easily do <button role="link"> but I think that may be breaking some rules of aria usage (don't change native semantics) and then links only work with enter key but buttons also work with spacebar so it may be weird.

**Comment by antelopeb** [comment]
@pauljadam  & @dengeist -- can you take one more look? Thanks!!

**Pull request merged**